### ISSUE TITLE
[Bug]: Game crash while in solo play

### ISSUE THREAD

**Issue by NeoConker2k22** [LukeYui/EldenRingSeamlessCoopRelease#240]
### Describe the issue in detail

I had the map open for ~2 minutes deciding on where to go next. I closed the map, didn't teleport anywhere, and the game image froze with half of the inventory window open but I could still hear the audio of my character moving if I pressed on the thumbstick to move. After about 5-10 seconds of this the game then crashed to desktop.

I was playing the game solo without an active P2P session open (did not use Tiny Great Pot) at the time the crash occurred. Also, all games files validated successfully prior to this crash (I do so after every update) -- though I verified them again to be sure and all files validated successfully.

### Steps to recreate the bug

Please see description and crash log as I'm not sure this is something that's easily repeatable as I've performed this action dozens of times before without crashes.

### Version

1.2.4 (Latest)

### Relevant Crash log

```shell
Faulting application name: eldenring.exe, version: 1.5.0.0, time stamp: 0x6295c83c
Faulting module name: nvwgf2umx.dll, version: 31.0.15.1640, time stamp: 0x629e9005
Exception code: 0xc0000005
Fault offset: 0x0000000000d4bdc8
Faulting process id: 0x1dcc
Faulting application start time: 0x01d8826a812da016
Faulting application path: C:\Program Files (x86)\Steam\steamapps\common\ELDEN RING\Game\eldenring.exe
Faulting module path: C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_c43eff7079c4c90c\nvwgf2umx.dll
Report Id: 301d627a-c3cb-4cea-bbb0-db9f73d23e2e
Faulting package full name: 
Faulting package-relative application ID: 

- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name="Application Error" /> 
  <EventID Qualifiers="0">1000</EventID> 
  <Version>0</Version> 
  <Level>2</Level> 
  <Task>100</Task> 
  <Opcode>0</Opcode> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2022-06-17T17:20:48.4863395Z" /> 
  <EventRecordID>2328</EventRecordID> 
  <Correlation /> 
  <Execution ProcessID="13580" ThreadID="0" /> 
  <Channel>Application</Channel> 
  <Computer>DESKTOP-COMPUTER</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>eldenring.exe</Data> 
  <Data>1.5.0.0</Data> 
  <Data>6295c83c</Data> 
  <Data>nvwgf2umx.dll</Data> 
  <Data>31.0.15.1640</Data> 
  <Data>629e9005</Data> 
  <Data>c0000005</Data> 
  <Data>0000000000d4bdc8</Data> 
  <Data>1dcc</Data> 
  <Data>01d8826a812da016</Data> 
  <Data>C:\Program Files (x86)\Steam\steamapps\common\ELDEN RING\Game\eldenring.exe</Data> 
  <Data>C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_c43eff7079c4c90c\nvwgf2umx.dll</Data> 
  <Data>301d627a-c3cb-4cea-bbb0-db9f73d23e2e</Data> 
  <Data /> 
  <Data /> 
  </EventData>
  </Event>
```

**Comment by Devologger** [comment]
As specified in the crash log, the faulting module is nvwgf2umx.dll which indicates that its a GPU / Nvidia Driver problem. Have you tried the game without the mod and see if it crashes?

**Comment by NeoConker2k22** [comment]
> As specified in the crash log, the faulting module is nvwgf2umx.dll which indicates that its a GPU / Nvidia Driver problem. Have you tried the game without the mod and see if it crashes?

I should have researched this crash myself, so I appreciate you bringing it to my attention that this was a display driver issue. I had just updated to the latest Nvidia drivers so it may have been related. I've since reinstalled my display drivers and I hope that resolved the issue. I assume this crash was unrelated to the mod so I'll close this bug.
